test/parser/parserize.ml

```diff
@@ -1,29 +1,35 @@
 open Ast 
 
-let rec eval = function
+let txt_of_op = function
```

Comment:
Do we need operations for anything other than binop? If not, I'm ok with removing this and simplifying the generator. If so, we definitely should keep this around.